servers/quarkus-server/src/main/java/org/projectnessie/server/filters/SinglePageAppRoutingFilter.java

```diff
@@ -23,7 +23,7 @@
 
   // maintain list of base paths that should serve root html.
   // must be updated as more UI paths are introduced
-  private static final String[] UI_ROUTES = {"/tree"};
+  private static final String[] UI_ROUTES = {"/tree", "/content", "/notfound"};
```

Comment:
we should add `Config` resource and `Diff` resource also?

Reply:
Why? Where's "diff" in the UI?

Reply:
yes, we don't have `Diff` feature in UI. It is only supported from REST API.